charts/kruise/v0.6.0/templates/webhookconfiguration.yaml

```diff
@@ -180,6 +180,24 @@ webhooks:
           - UPDATE
         resources:
           - clonesets
+  - clientConfig:
```

Comment:
Should not edit chart template, this will be added in next release

Reply:
ok, removed